Tests/Common/Securities/Equity/PatternDayTradingMarginModelTests.cs

```diff
@@ -0,0 +1,221 @@
+            TimeZones.NewYork);
+
+        private static Security CreateSecurity(DateTime newLocalTime)
+        {
+            var security = new Security(CreateUsEquitySecurityExchangeHours(), CreateTradeBarConfig());
+            security.Exchange.SetLocalDateTimeFrontier(newLocalTime);
+            security.SetLocalTimeKeeper(TimeKeeper.GetLocalTimeKeeper(TimeZones.NewYork));
+            security.SetMarketPrice(new IndicatorDataPoint(Symbols.SPY, newLocalTime, 100m));
+            return security;
+        }
+
+        private static SecurityExchangeHours CreateUsEquitySecurityExchangeHours()
+        {
+            var sunday = LocalMarketHours.ClosedAllDay(DayOfWeek.Sunday);
+            var monday = new LocalMarketHours(DayOfWeek.Monday, new TimeSpan(9, 30, 0), new TimeSpan(16, 0, 0));
+            var tuesday = new LocalMarketHours(DayOfWeek.Tuesday, new TimeSpan(9, 30, 0), new TimeSpan(16, 0, 0));
+            var wednesday = new LocalMarketHours(DayOfWeek.Wednesday, new TimeSpan(9, 30, 0), new TimeSpan(16, 0, 0));
+            var thursday = new LocalMarketHours(DayOfWeek.Thursday, new TimeSpan(9, 30, 0), new TimeSpan(16, 0, 0));
+            var friday = new LocalMarketHours(DayOfWeek.Friday, new TimeSpan(9, 30, 0), new TimeSpan(16, 0, 0));
+            var saturday = LocalMarketHours.ClosedAllDay(DayOfWeek.Saturday);
+
+            return new SecurityExchangeHours(TimeZones.NewYork, USHoliday.Dates.Select(x => x.Date), new[]
+            {
+                sunday, monday, tuesday, wednesday, thursday, friday, saturday
+            }.ToDictionary(x => x.DayOfWeek));
+        }
+
+        private static SubscriptionDataConfig CreateTradeBarConfig()
+        {
+            return new SubscriptionDataConfig(typeof (TradeBar), Symbols.SPY, Resolution.Minute, TimeZones.NewYork,
+                TimeZones.NewYork, true, true, false);
+        }
+
+        [Test]
+        public void InitializationTests()
+        {
+            // No parameters initialization, used default PDT 4x leverage open market and 2x leverage otherwise
+            var model = new PatternDayTradingMarginModel();
+            var leverage = model.GetLeverage(CreateSecurity(Noon));
+
+            Assert.AreEqual(4.0m, leverage);
+
+            model = new PatternDayTradingMarginModel(2.0m, 5.0m);
+            leverage = model.GetLeverage(CreateSecurity(Noon));
+
+            Assert.AreEqual(5.0m, leverage);
+        }
+
+        [Test]
+        public void VerifyClosedMarketLeverage()
+        {
+            // Market is Closed on Tuesday, Feb, 16th 2016 at Midnight
+
+            var leverage = 2m;
+
+            var model = new PatternDayTradingMarginModel();
+            var security = CreateSecurity(MidNight);
+            var order = new MarketOrder(security.Symbol, 100, security.LocalTime, type: security.Type);
+
+            var expected = 100*100m/leverage + 1;
+            var actual = model.GetInitialMarginRequiredForOrder(security, order);
```

Comment:
I wonder what the result would be if I called `model.GetLeverage()` here -- I would expect to get 2x, correct?